### DISCUSSION TITLE
2024-08-15-r1

### DISCUSSION THREAD

**Discussion by Zoey2936** [ZoeyVid/NPMplus#1015] [Announcements]
### Changes
- dep updates
- fix healthcheck #1014 

### How to update
- Read the changes above
- Pull the zoeyvid/npmplus:latest image
- apply possible changes that maybe effect you from above to your compose.yaml/NPMplus
- redeploy the compose stack

**Full Changelog**: https://github.com/ZoeyVid/NPMplus/compare/2024-08-03-r1...2024-08-15-r1

<hr /><em>This discussion was created from the release <a href='https://github.com/ZoeyVid/NPMplus/releases/tag/2024-08-15-r1'>2024-08-15-r1</a>.</em>

**Comment by hongmd** [comment]
Waiting for the new update :D

**Reply by Zoey2936** [reply]
#1103 still needs to be fixed
